doc/tools/make_rst.py

```diff
@@ -1261,7 +1303,11 @@ def make_method_signature(
     return ret_type, out
 
 
-def make_heading(title, underline):  # type: (str, str) -> str
+def make_heading(title, underline, l10n=True):  # type: (str, str, bool) -> str
+    if l10n:
+        if title in headings_l10n:
+            title = headings_l10n.get(title)
+            underline *= 2  # Double length to handle wide chars.
```

Comment:
Maybe it should be something like `(underline * len(title.decode('utf-8')))` to get the actual number of Unicode characters (not sure if it's what it wants).

Reply:
`title` is a `str` in Python 3 I believe, so no need/way to decode 😛

Reply:
It's actually the visual width in monospace columns that we want, but that's tricky to get, see https://github.com/godotengine/godot/pull/56127/files/09563dc665dbda970b84f5c63955155b8f8857e6#diff-8cfde96f29054d38628a87af04b23d77280174c3d31df0c38a23cbd34bce960a